examples/complex_module/README.md

```diff
@@ -44,15 +46,15 @@ An example configuration for an Arm component, a Gizmo component, and a Summatio
     {
       "name": "arm1",
       "type": "arm",
-      "model": "acme:demo:myarm",
+      "model": "viam:arm:myarm",
```

Comment:
you have to make the corresponding changes in the actual class definition as well, same for all the other example models